docs/tempo/website/getting-started/_index.md

```diff
@@ -19,7 +22,7 @@ out these pieces, it may look something like this:
 
 ## 1. Instrumentation
```

Comment:
Can we take the numbers out of the heading, and instead consolidate steps?

Reply:
Done.